profile/src/wallet/secure_storage/secure_storage.rs

```diff
@@ -0,0 +1,8 @@
+use crate::prelude::*;
+
+#[uniffi::export]
+pub trait SecureStorage: Send + Sync + std::fmt::Debug {
+    fn load_data(&self, key: SecureStorageKey) -> Result<Option<Vec<u8>>>;
```

Comment:
Non final API... might have to change to be `async` due to limitations by Android client?